README.md

```diff
@@ -1,42 +1,44 @@
-# tikv-jemallocator
+# jemallocator
 
 [![ci]][github actions] [![Latest Version]][crates.io] [![docs]][docs.rs]
 
 This project is a simplified fork of [jemallocator](https://github.com/gnzlbg/jemallocator) focus on server.
```

Comment:
As this repo will be the successor of gnzlbg/jemallocator, I think we should change this sentence to clarify the status of this repo.